infracost-usage-example.yml

```diff
@@ -9,6 +9,17 @@ resource_type_default_usage:
     request_duration_ms: 500 # Average duration of each request in milliseconds.
 resource_usage:
   #
+  # Cloudformation AWS resources
```

Comment:
I'm going to remove this example example for just now. We're currently reworking some of the usage stuff. If we have examples for CloudFormation in the future we should keep them separate from the Terraform ones.